_test/test_rebin/test_rebin_operations.m

```diff
@@ -145,17 +141,15 @@
             mf2 = mf2.set_fun (@fake_cross_sec, [80,0.1,10]);
             [~, fitdata2] = mf2.fit();
 
-            [ok,mess]=equal_to_tol(fitdata1.p,fitdata2.p,-5e-3,'ignore_str', 1);
-            assertTrue(ok,['rebin dnd using lo,step,hi syntax fails: ',mess])
+            assertEqualToTol(fitdata1.p,fitdata2.p,-5e-3,'ignore_str', 1);
```

Comment:
this gets rid of ok mess but also removes comment on the mode of failure tested. worth putting the message into a comment, also for LL.190ff